### ISSUE TITLE
branch 2.1.0 servicing package validation error.

### ISSUE THREAD

**Issue by StephenBonikowsky** [dotnet/wcf#2909]
With https://github.com/StephenBonikowsky/wcf/tree/stebon/2.1.0/UpdatePkgAndAssembliesForServicing I am moving our release/2.1.0 branch to servicing.

After making the necessary changes I am getting a package validation error only for System.ServiceModel.Security.

The errors are...
```
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.1 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net461" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.1/win-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net461" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.1/win-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net461" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.1/win7-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net461" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.1/win7-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net461" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.2 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net462" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.2/win-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net462" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.2/win-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net462" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.2/win7-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net462" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.6.2/win7-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net462" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.7 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net47" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.7/win-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net47" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.7/win-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net47" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.7/win7-x86 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net47" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
C:\OSS\wcf\Tools\Packaging.targets(1131,5): error : System.ServiceModel.Security should be supported on .NETFramework,Version=v4.7/win7-x64 but has a compile placeholder.  You may need to remove InboxOnTargetFramework Include="net47" /> from your project. [C:\OSS\wcf\src\System.ServiceModel.Security\pkg\System.ServiceModel.Security.pkgproj]
```

**Comment by StephenBonikowsky** [comment]
@ericstj could you take a look? The error message says to remove InboxOnTargetFramework for several frameworks but they don't exist in the pkgproj.

**Comment by ericstj** [comment]
/cc @joperezr 
You increased the assembly version but you're still claiming its inbox.  What would happen if someone compiled against netstandard2.0 and then tried to run it on desktop?  Perhaps you should just test that out if you can't make sense of these errors.

**Comment by StephenBonikowsky** [comment]
The error message about removing "InboxOnTargetFramework", was misleading.
We hadn't realized that this was the first time the minor version of the assembly had increased since moving to open source therefore becoming "outofbox" for the first time and requiring the package to now also build a netfx facade.

**Comment by StephenBonikowsky** [comment]
Fixed with #2910
